### DISCUSSION TITLE
Option to see daily table of all birds (not just top and bottom 10)

### DISCUSSION THREAD

**Discussion by waynecarnold** [mcguirepr89/BirdNET-Pi#958] [Ideas]
Would love an option to see a chart of all birds detected per day, not just the top 10 table or bottom 10.

Thanks
Wayne

**Comment by Svardsten53** [comment]
I agree, never quite understood why only the ten most frequent species are shown.

However, I have a modified version, which just creates an overview with all the species and you are welcome to use it.
[daily_plot.txt](https://github.com/mcguirepr89/BirdNET-Pi/files/11781266/daily_plot.txt)

Rename attached file to daily_plot.py and replace the file /scripts/daily_plot.py. You have to change the text "change-userid" to your own id in two places and also the two title lines.

**Reply by deepcoder** [reply]
I believe if you let BirdNET-Pi do it's update, this will overwrite your daily_plot.py file.

**Comment by Svardsten53** [comment]
Yes, you have to replace the file after every update.
